types/client/channel.ts

```diff
@@ -0,0 +1,11 @@
+import { object, TypeOf, z } from 'zod';
+
+export const createChannelSchema = object({
+  name: z
+    .string({ required_error: 'Channel name is required' })
+    .min(3, { message: 'Must be at least 3 characters' }),
```

Comment:
Can we validate this to `.trim()` and ensure spaces just spaces aren't allowed? As it stands I can enter `   ` and that's an acceptable channel name.

Reply:
Note this has been mirrored on Postgres side now as well.